flink-libraries/flink-table/src/main/scala/org/apache/flink/table/factories/TableFactory.scala

```diff
@@ -16,14 +16,14 @@
  * limitations under the License.
  */
 
-package org.apache.flink.table.connectors
+package org.apache.flink.table.factories
 
 import java.util
 
 /**
   * Common trait for all properties-based discoverable table factories.
   */
-trait DiscoverableTableFactory {
+trait TableFactory {
```

Comment:
Actually, I would like the very generic name `Factory` but since we have to add some prefix to make it unique in the project, I named it `TableFactory` because we prefix everything in this module with `Table`.

Reply:
I am not happy but I can not think about something better :( `Factory` is way to generic. It implies that this is THE Factory, THE One Factory To Rule Them All and that's not true. There are plenty of other factories in `flink-table`. `Table` prefix as you confirmed is just a package prefix, thus doesn't improve anything :(